pages/docs/tutorials/kafka/index.md

```diff
@@ -0,0 +1,188 @@
+---
+title: Create AsyncAPI Document for applications consuming from Kafka
+description: Explore configuring AsyncAPI document for Kafka messages.
+weight: 20
+---
+
+## Introduction
+
+The tutorial will walk you through the process of creating a schema document for Kafka Messages using AsyncAPI. Additionally, you will also learn about event-driven architecture, pub/sub model, and brokers in relation to Kafka. 
+  
+Let’s assume, you have a service that publishes data to a Kafka topic whenever a new user signs up into the application.
+
+Basically, if you want to build a system that can track the users who have signed up for your application, the entire flow of the application would be:
+
+```mermaid
+sequenceDiagram
+    participant Your Application (Consumer)
+    participant Kafka
+    participant Service (Producer)
+    participant User
+    Your Application (Consumer)->> Kafka: Subscribes to "userSignedUp" topic
+    User-->> Service (Producer): Signs Up
+    Service (Producer)->> Kafka: Publishes Events
+    Kafka->> Your Application (Consumer): Sends "user_signup" message
+```    
+
+
+## Background context
+
+[Event-driven architecture (EDA)](/docs/tutorials/getting-started/event-driven-architectures) is a design pattern that revolves around the production, detection, and reaction to events over time. It consists of three main components: a message broker, event publishers, and subscribers, which together serve as the backbone for event exchange within different services. 
+
+[Message brokers](/docs/tutorials/getting-started/event-driven-architectures#message-broker) facilitate asynchronous communications between services, meaning that the sending service doesn't have to wait for the receiving service's response. This allows multiple services to talk to each other directly, even if they are written in different languages. A great example of such a message broker is Apache Kafka. [Apache Kafka](https://kafka.apache.org/) is a distributed system that is used by thousands of companies for their event-driven system.
```

Comment:
> shouldn't there be some intro to Kafka?

@derberg I have already defined Kafka while defining Message Broker, should I add more introduction?

Reply:
might be ok, leaving it up to you and other writers

with my experience I might be biased

Reply:
Personally I think this much content is enough, but I'll try checking with others

Reply:
I actually liked how you seamlessly defined message brokers and then used that as the starting point to define Kafka next.

That said, I also agree with Lukasz that it initially feels as if we "forgot" to also introduce/define Kafka. I would vote to move your Kafka sentences into their own paragraph, so it doesn't get lost in the message broker definition.  I would also consider if you can add a sentence to describe what is a "distributed system" because it's not clear how that relates to the term "message brokers". Lemme know if that makes sense 😄

Reply:
Yeah, it makes sense. I'll make the changes ASAP